tests/src/cliruntimefixture.cpp

```diff
@@ -5,37 +5,47 @@
 
 #include "Luau/Compiler.h"
 
-static int capture(lua_State* L)
+static int report(lua_State* L)
 {
     const char* str = luaL_tolstring(L, 1, nullptr);
-    lua_pushstring(L, "capturedoutput");
-    lua_pushstring(L, str ? str : "");
-    lua_settable(L, LUA_REGISTRYINDEX);
+    lua_getglobal(L, "reporter");
+    if (lua_islightuserdata(L, -1))
+    {
+        // It's a light userdata, proceed to retrieve
+        void* rep = lua_touserdata(L, -1); // Get it from the top of the stack
+        TestReporter* reporter = static_cast<TestReporter*>(rep);
+        reporter->reportOutput(str);
+    }
+    else
+    {
+        throw std::runtime_error("Expected reporter to be a light userdata");
+    }
```

Comment:
```suggestion
    lua_getfield(L, LUA_REGISTRYINDEX, "reporter");
    TestReporter* reporter = static_cast<TestReporter*>(lua_touserdata(L, -1));
    reporter->reportOutput(str);
```
`lua_touserdata` will return `nullptr` if it's not the correct type, so this will segfault if we've set it up wrong. We shouldn't need to do runtime checks for this since the Luau API guarantees it will work if we've set it up correctly in the `setupCliState` callback.

Reply:
fixed!